tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -17,6 +17,7 @@
 <io-gateway-capture.conf>
 <host-copier-gain-mixin-playback.conf>
 <mixout-gain-dai-copier-playback.conf>
+<mixout-gain-eqiir-dts-dai-copier-playback.conf>
```

Comment:
I guess this is referring to a file that is not yet available in the tree (added in the next patch), so this would break git bisect (e.g. after each git commit, the tree should build and be functional).

Reply:
This file is added in the first patch(b18b769ac2a81067d268fe08831e92e62ead085f). It shouldn't break the build. But it makes sense to move it to next patch.